src/main/java/seedu/address/logic/commands/DeleteCommand.java

```diff
@@ -18,36 +23,77 @@
     public static final String COMMAND_WORD = "delete";
 
     public static final String MESSAGE_USAGE = COMMAND_WORD
-            + ": Deletes the person identified by the index number used in the displayed person list.\n"
-            + "Parameters: INDEX (must be a positive integer)\n"
-            + "Example: " + COMMAND_WORD + " 1";
+            + ": Deletes the person identified by the index number used in the displayed person list "
+            + "or matriculation number.\n"
+            + "Parameters: INDEX (must be a positive integer) or STUDENT_ID\n"
+            + "Example: " + COMMAND_WORD + " 1"
+            + " or " + COMMAND_WORD + " " + PREFIX_ID + "A0123456Z \n";
```

Comment:
```suggestion
            + " or " + COMMAND_WORD + " " + PREFIX_ID + "A0123456Z\n";
```

Just a minor issue. Not sure if you intended for this extra whitespace.

Reply:
Thanks for the catch 👍